packages/core/test.setup.js

```diff
@@ -0,0 +1,7 @@
+/* globals module require globalThis */
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
We should rename this file to use `.cjs`.  I believe there is a PR open to change `core` to `"type": "module"`